docs/user/overview.md

```diff
@@ -0,0 +1,61 @@
+
+  1. The bootstrap machine boots and starts hosting the remote resources required for the master machines to boot.
+  2. The master machines fetch the remote resources from the bootstrap machine and finish booting.
+  3. The master machines use the bootstrap node to form an etcd cluster.
+  4. The bootstrap node starts a temporary Kubernetes control plane using the newly-created etcd cluster.
+  5. The temporary control plane schedules the long-running control plane to the master machines.
+  6. The temporary control plane shuts down, yielding to the long-running control plane.
+  7. The bootstrap node injects OpenShift-specific components into the newly formed control plane.
+  8. The installer then tears down the bootstrap node.
+
+The result of this bootstrapping process is a fully running OpenShift control plane. The cluster will then download and configure remaining components needed for the day-to-day operation, including the creation of worker machines in supported environments.
+
+[ignition]: https://github.com/coreos/ignition/blob/master/doc/getting-started.md
+
+
+## Key Concepts ##
+
+While striving to remain simple and easy to use, the installer allows many aspects of the clusters it creates to be customized. It is helpful to understand certain key concepts before attempting to customize the installation.
+
+
+### Targets ###
+
+The OpenShift Installer operates on the notion of creating and destroying targets. Similar to other tools which operate on a graph of dependencies (e.g. make, systemd), each target represents a subset of the dependencies in the graph. The main target in the installer creates a cluster, but the other targets allow the user to interrupt this process and consume or modify the intermediate artifacts (e.g. the Kubernetes manifests that will be installed into the cluster). Only the immediate dependencies of a target are written to disk by the installer, but the installer can be invoked [multiple times](#multiple-invocations).
+
+The following targets can be created by the installer:
+
+- `install-config` - The install config contains all of the parameters for the installation process. This configuration provides the user with more options than the interactive prompts and comes pre-populated with default values.
+- `manifests` - This target outputs all of the Kubernetes manifests that will be installed on the cluster.
+- `ignition-configs` - These are the three Ignition Configs for the bootstrap, master, and worker machines.
+- `cluster` - This actually provisions the cluster by invoking the locally-installed Terraform.
+
+The following targets can be destroyed by the installer:
+
+- `cluster` - This destroys the created cluster and its associated infrastructure (if applicable).
```

Comment:
What is "(if applicable)" about?  This destroys the cluster and its infrastructure.  If you run it on an AWS cluster you created yourself, it will still tear it down assuming you set the right tags ;).